modules/framegrabber/include/ifm3d/fg/byte_buffer.h

```diff
@@ -342,6 +343,15 @@ namespace ifm3d
      */
     float IlluTemp();
 
+    /**
+     * Returns JSON model of the output of active application
+     *
+     * NOTE: To get the JSON data of the application running on the device, you need
+     * to make sure your current pcic schema asks for it by using ifmed::JSON_MODEL in schema.
+     * this will return "{}" blank JSON string for camera devices.
```

Comment:
@inbangsa , My O3D303 returns a populated JSON string, not an empty string. I can update the comment in my PR for the python bindings, but can you please clarify the expected behavior?